apps/gitness/src/framework/context/AppContext.tsx

```diff
@@ -3,13 +3,18 @@ import {
   CodeServiceAPIClient,
   MembershipSpacesOkResponse,
   TypesMembershipSpace,
-  membershipSpaces
+  membershipSpaces,
+  TypesSpace
 } from '@harnessio/code-service-client'
 
 interface AppContextType {
   spaces: TypesMembershipSpace[]
   setSpaces: (spaces: TypesMembershipSpace[]) => void
+  addSpace: (newSpaces: TypesSpace[]) => void
```

Comment:
nit: typo `newSpace` instead of `newSpaces`.

Reply:
Fixed